components/gitpod-protocol/src/usage.ts

```diff
@@ -24,10 +24,10 @@ export interface BillableSession {
     workspaceClass: string;
 
     // When the workspace started
-    startTime: number;
+    startTime: string;
 
     // When the workspace ended
-    endTime: number;
+    endTime: string;
```

Comment:
I did this to follow the pattern we have going in virtually every other place in our JSON API. :+1:

Reply:
I think this is fine. The gRPC protocol will use `google.Time` which can be converted into a string easily